src/components/UserInfo.js

```diff
@@ -16,4 +18,8 @@ export default class UserInfo {
     this._profileName.textContent = name;
     this._profileJob.textContent = job;
   }
+
+  setUserAvatar(avatar) {
```

Comment:
It's great to have a separate method for updating the avatar as it's controlled by a different form